lib/game_team_manager.rb

```diff
@@ -157,4 +157,20 @@ def goals(games)
   def games(games)
     games.count
   end
+
+  def average_win_percentage(team_id)
```

Comment:
can you rename this argument being passed its the same as the attr reader for the object so just for clarity